docs/DesignTokens/Taxonomy.stories.mdx

```diff
@@ -0,0 +1,113 @@
+import { Meta } from "@storybook/addon-docs";
+
+<Meta title="Design Tokens Taxonomy" />
+
+# Design Tokens Taxonomy — Work in Progress 🛠
+
+🛠 **Work in Progress** — We're iterating and refining the conventions stated here as we work on authoring our design system tokens and components.
+
+## Token tiers
+
+Our design tokens are conceptually organized into three tiers — base, semantic, and component tokens. The top tier is global and unaware of the intended context of use, while subsequent tiers are aware of context and increasingly more precise about it.
+
+- **Base tokens** — Unaware of the intended context of use.
+- **Semantic tokens** — Aware of context and shared across components.
+- **Component tokens** — Aware of context but specific to one component.
+
+With this token architecture, we can update design decisions globally or target changes to a single component.
+
+### Base
+
+Base tokens enumerate the available values in the system, without providing specific directions on how or where to use them. Used for global, literal values. 
+
+Example: `cds.color.blue.800`, which holds the value `#004ce6`.
+
+### Semantic
+
+Semantic tokens describe the purpose of tokens in specific contexts. They are aliases to base tokens that add meaning about the different possible use cases for base tokens, by describing where or how they are used. This way we can target each use case indepedently.
+
+Example: `cds.color.background.primary`, which points to `cds.color.blue.800`.
```

Comment:
We need to update all our tokens to use `cds`, or we need to document that as `cpd` and change it at a later stage. 
Because this does not match the reality

Reply:
Reverted it to `cpd.`